projects/organization-management/src/app/services/users/users.service.spec.ts

```diff
@@ -35,4 +36,16 @@ describe('Users Service', () => {
       done();
     });
   });
+
+  it('should call delete method of customer API when delete user', done => {
```

Comment:
tests for addUser and updateUser are missing, why is delete user in this branch?

Reply:
this was the branch that was discussed for me to use, since already had some of features i would probably need